topo/node/cisco/cisco.go

```diff
@@ -413,6 +423,13 @@ func defaults(pb *tpb.Node) (*tpb.Node, error) {
 		if pb.Config.Image == "" {
 			pb.Config.Image = "8000e:latest"
 		}
+		if pb.HostConstraints == nil {
```

Comment:
hostconstraints will never be nil at this point

Reply:
Fixed.